azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/ChatComposite.java

```diff
@@ -3,5 +3,141 @@
+                       final ChatCompositeLocalOptions localOptions) {
+        launchComposite(context, remoteOptions, localOptions, false);
+    }
+
+    /**
+     * Stop the ChatComposite. Destroy the UI if in foreground mode. Destroy service layer.
+     */
+    public void stop() {
+        chatContainer.stop();
+    }
+
+    private void launchComposite(final Context context,
+                                 final ChatCompositeRemoteOptions remoteOptions,
+                                 final ChatCompositeLocalOptions localOptions,
+                                 final boolean isTest) {
+    }
+
+    /**
+     * Get Composite UI view
+     *
+     * @param context The android context used to start the Composite.\
+     * @return View ChatComposite UI view
+     */
+    public View getCompositeUIView(final Context context) {
+        return null;
+    }
+
+    /**
+     * To show full composite default view
+     *
+     * @param context The android context used to start the Composite.
+     */
+    public void showCompositeUI(final Context context) {
+    }
+
+    /**
+     * To hide full composite view
+     *
+     * @param context The android context used to start the Composite.
+     */
+    public void hideCompositeUI(final Context context) {
+    }
+
+    /**
+     * Add {@link ChatCompositeEventHandler}.
+     *
+     * @param handler The {@link ChatCompositeEventHandler}.
+     */
+    public void addOnViewClosedEventHandler(final ChatCompositeEventHandler<Object> handler) {
+    }
+
+    /**
+     * Remove {@link ChatCompositeEventHandler}.
+     *
+     * @param handler The {@link ChatCompositeEventHandler}.
+     */
+    public void removeOnViewClosedEventHandler(final ChatCompositeEventHandler<Object> handler) {
+    }
+
+    /**
+     * Add {@link ChatCompositeEventHandler} with {@link ChatCompositeUnreadMessageChangedEvent}.
+     *
+     * @param handler The {@link ChatCompositeEventHandler}.
+     */
+    public void addOnUnreadMessagesChangedEventHandler(
+            final ChatCompositeEventHandler<ChatCompositeUnreadMessageChangedEvent> handler) {
+    }
+
+    /**
+     * Remove {@link ChatCompositeEventHandler} with {@link ChatCompositeUnreadMessageChangedEvent}.
+     *
+     * @param handler The {@link ChatCompositeEventHandler}.
+     */
+    public void removeOnUnreadMessagesChangedEventHandler(
+            final ChatCompositeEventHandler<ChatCompositeUnreadMessageChangedEvent> handler) {
+
+    }
+
+    /**
+     * Set {@link ChatCompositeRemoteOptions}.
+     *
+     * <p>
+     * Used to set Participant View Data (E.g. Avatar and displayName) to be used on this device only.
+     * </p>
+     *
+     * @param participantViewData The {@link ChatCompositeRemoteOptions}.
+     * @param identifier          The {@link CommunicationIdentifier}.
+     */
+    public void setChatCompositeRemoteOptions(ChatCompositeRemoteOptions participantViewData, final CommunicationIdentifier identifier) {
```

Comment:
Do we need this method?
If Contoso already called launch method, is it too late to try to set RemoteOptions